test/index.spec.js

```diff
@@ -0,0 +1,27 @@
+import { init } from '../src'
+import Enzyme, { mount } from 'enzyme'
+import Adapter from 'enzyme-adapter-react-16'
+
+Enzyme.configure({ adapter: new Adapter() })
+
+describe('The bar library', function () {
+  it('should render correctly the cozy-bar', () => {
+    // Set up our document body
+    document.body.innerHTML =
+      '<div role="application">' +
+      '  The application will be here' +
+      '</div>'
+    const options = {
+      appName: 'test-app',
+      appEditor: 'cozy',
+      lang: 'en',
+      iconPath: '',
+      cozyURL: 'https://mock.cozy',
+      token: 'mock',
+      displayOnMobile: true
+    }
+    const Bar = init(options)
+    const mounted = mount(Bar)
+    expect(mounted.getElement()).toMatchSnapshot()
+  })
```

Comment:
Can you add a test when you change the language ?

Reply:
I didn't find a good way to test that case here for now. The mounted bar here doesn't give me a full rendered bar as we could expect. So there is no way to get the difference between a lang an another.

Reply:
What is missing in the rendered component here is only the lang ? It seems in the test that the store has `lang: undefined`

Reply:
I was trying to test directly the rendered parts or components. I don't wanted to test directly the store since it will repeat [that store test](https://github.com/cozy/cozy-bar/blob/master/test/lib/api.spec.js#L47) I think.

Reply:
ok